README.md

```diff
@@ -103,12 +94,13 @@ After executing `atlasreader` on a given image, four kinds of outputs are genera
 - **stat_plot_kws**: dict or None, optional  
     Additional keyword arguments to pass to `nilearn.plotting.plot_stat_map`.
 
-For a more detailed explanation about the toolbox and the effect of the parameters above, see the [example notebook](https://github.com/miykael/atlasreader/blob/master/notebooks/atlasreader.ipynb).
+For a more detailed explanation about the toolbox and the effect of the
+parameters above, see the [example notebook](https://github.com/miykael/atlasreader/blob/master/notebooks/atlasreader.ipynb).
 
 
 ## Installation
```

Comment:
I was expecting installation above the usage....I'm going to leave this comment here while I keep reading....

Reply:
Yes, makes perfect sense. Just changed that with https://github.com/miykael/atlasreader/pull/66/commits/334512a8662dd6e14ec8973d5b081fa76f6c0e7e.